opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -285,12 +281,11 @@ namespace Opm {
             std::vector<double> residual_norms;
             perfTimer.reset();
             perfTimer.start();
-            report.converged = getConvergence(timer, iteration,residual_norms);
+            // the step is not considered converged until at least minIter iterations is done
```

Comment:
Why do you need to know the reason for taking more iterations?
Adding a member "finished" in the simulatorReport would make it possible to distinguish the reason. It is a bit dangerous since most people would assume that converged == finished.